executor/cook/executor.py

```diff
@@ -420,13 +420,16 @@ def reregistered(self, driver, agent_info):
         if self.config.checkpoint:
             logging.info('Executor checkpointing is enabled. Notifying on reregister_signal')
             self.reregister_signal.set()
```

Comment:
We should guard this and the next statement inside a `self.reregister_signal is not None` if clause with a log statement on the else branch.

Reply:
Done